flake.nix

```diff
@@ -0,0 +1,32 @@
+{
+  description = "A shading language";
```

Comment:
Use something like this: "The Slang Shading Language and Compiler"?

Reply:
@saipraveenb25 done!